frontend/src/components/management/WorkflowAssignment.tsx

```diff
@@ -83,7 +83,8 @@ export function WorkflowAssignment(props: LabworkSamplesProps) {
                 render: (_, { sample }) => {
                     if (!sample) return null
                     const sampleNextSteps = sampleNextStepsBySampleID[sample.id]
-                    if (!sampleNextSteps || sampleNextSteps.length === 0) return null
+                    if (!sampleNextSteps) return <Spin size={"small"} />
+                    if (sampleNextSteps.length === 0) return null
```

Comment:
Is this case actually happening ? sampleNextStepsBySampleID is likely to never return an empty list since it is built from queued samples ...

Reply:
Oh I see. It would return `undefined` if there is no sample next step.